system-addon/lib/SectionsManager.jsm

```diff
@@ -34,7 +34,26 @@ const BUILT_IN_SECTIONS = {
     emptyState: {
       message: {id: "topstories_empty_state", values: {provider: options.provider_name}},
       icon: "check"
-    }
+    },
+    order: 0
+  }),
+  "feeds.section.highlights": options => ({
+    id: "highlights",
+    pref: {
+      titleString: {id: "settings_pane_highlights_header"},
+      descString: {id: "settings_pane_highlights_body2"}
```

Comment:
I'm guessing section manager doesn't currently support the complex sub pref options, so we can split that out to fully fix #3155.